index.html

```diff
@@ -17,31 +17,44 @@
 </head>
 
 <body>
-  <div class="container-fluid">
-    <div class="container">
+  <div class="container sticky-top">
```

Comment:
Should have remained `container-fluid` I know we talked about this and you were unable to make the internal elements work properly. It's on my notes to research more about.